Edit Resolving Package Dependencies (#2300)
theforeman/foreman-documentation: guides/common/modules/proc_enabling-dependency-solving-globally.adoc

[id="enabling-dependency-solving-per-content-view_{context}"]
= Enabling Dependency Solving Per Content View

> You can enable dependency solving for a Content View or by default for your whole {Project} environment. Use this procedure to enable it for a Content View.

Reviewer: ```suggestion
You can enable dependency solving for a Content View or by default for your whole {Project} environment.
Use this procedure to enable it for a Content View.
```

.Prerequisite

* Dependency solving is useful only in limited contexts.

> Before enabling it, ensure you read and understand xref:Resolving_Package_Dependencies_content-management[]

Reviewer: ```suggestion
Before enabling it, ensure you read and understand xref:Resolving_Package_Dependencies_{context}[]
```

.Procedure

. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
. From the list of content views, select the required Content View.

> . Under *Details*, toggle *Solve dependencies*.

Reviewer 2: ```suggestion
. On the *Details*, toggle *Solve dependencies*.
```
I don't know if we are using `under`. Same applies below.
